src/CoreWCF.Primitives/src/CoreWCF/Channels/SecurityBindingElement.cs

```diff
@@ -298,6 +299,240 @@ private static BindingContext CreateIssuerBindingContextForNegotiation(BindingCo
             }
         }
 
+        // If any changes are made to this method, please make sure that they are
+        // reflected in the corresponding IsIssuedTokenOverTransportBinding() method.
+        static public TransportSecurityBindingElement CreateIssuedTokenOverTransportBindingElement(IssuedSecurityTokenParameters issuedTokenParameters)
+        {
+            if (issuedTokenParameters == null)
+                throw DiagnosticUtility.ExceptionUtility.ThrowHelperArgumentNull("issuedTokenParameters");
+
```

Comment:
nameof. Also add braces around if block. There's an intellicode helper to do that for the entire file.